rules/windows/privilege_escalation_port_monitor_registration.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
+creation_date = "2021/01/21"
+maturity = "production"
+updated_date = "2021/01/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+A port monitor can be configured to load a DLL at startup. An arbitrary DLL can be added to the Monitors registry path
+and executed as SYSTEM for privilege escalation or persistence, if permissions allow writing a fully-qualified pathname
+for that DLL.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Port Monitor Registration"
+risk_score = 47
+rule_id = "8f3e91c7-d791-4704-80a1-42c160d7aa27"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+registry where event.type in ("creation", "change") and
```

Comment:
bonus points from me -- if you make the user non-SYSTEM, you can basically guarantee that priv esc was happening, because the DLL will always get loaded by spoolsv (i think?) as SYSTEM, so you have a clear path from non-SYSTEM to SYSTEM.  that should help lower FPs and increase fidelity of the rule

Reply:
that s a great idea! ill add that in as well.